packages/manager/modules/at-internet-configuration/src/index.js

```diff
@@ -4,8 +4,6 @@ import '@ovh-ux/manager-core';
 import '@ovh-ux/ng-at-internet';
 import '@ovh-ux/ng-at-internet-ui-router-plugin';
 
-import { Environment } from '@ovh-ux/manager-config';
```

Comment:
Shouldn't be removed from `packages/manager/modules/at-internet-configuration/package.json` also?